Sources/SourceKitLSP/Documentation/DocumentationLanguageService.swift

```diff
@@ -0,0 +1,296 @@
+    // Nothing to set up
+  }
+
+  package func shutdown() async {
+    // Nothing to tear down
+  }
+
+  package func addStateChangeHandler(
+    handler: @escaping @Sendable (LanguageServerState, LanguageServerState) -> Void
+  ) async {
+    // There is no underlying language server with which to report state
+  }
+
+  package func openDocument(
+    _ notification: DidOpenTextDocumentNotification,
+    snapshot: DocumentSnapshot
+  ) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func closeDocument(_ notification: DidCloseTextDocumentNotification) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func reopenDocument(_ notification: ReopenTextDocumentNotification) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func changeDocument(
+    _ notification: DidChangeTextDocumentNotification,
+    preEditSnapshot: DocumentSnapshot,
+    postEditSnapshot: DocumentSnapshot,
+    edits: [SwiftSyntax.SourceEdit]
+  ) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func willSaveDocument(_ notification: WillSaveTextDocumentNotification) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func didSaveDocument(_ notification: DidSaveTextDocumentNotification) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func documentUpdatedBuildSettings(_ uri: DocumentURI) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func documentDependenciesUpdated(_ uris: Set<DocumentURI>) async {
+    // The DocumentationLanguageService does not do anything with document events
+  }
+
+  package func completion(_ req: CompletionRequest) async throws -> CompletionList {
+    .init(isIncomplete: false, items: [])
+  }
+
+  package func hover(_ req: HoverRequest) async throws -> HoverResponse? {
+    .none
+  }
+
+  package func symbolInfo(
+    _ request: SymbolInfoRequest
+  ) async throws -> [SymbolDetails] {
+    []
+  }
+
+  package func openGeneratedInterface(
+    document: DocumentURI,
+    moduleName: String,
+    groupName: String?,
+    symbolUSR symbol: String?
+  ) async throws -> GeneratedInterfaceDetails? {
+    nil
+  }
+
+  package func definition(
+    _ request: DefinitionRequest
+  ) async throws -> LocationsOrLocationLinksResponse? {
+    nil
+  }
+
+  package func declaration(
+    _ request: DeclarationRequest
+  ) async throws -> LocationsOrLocationLinksResponse? {
+    nil
+  }
+
+  package func documentSymbolHighlight(
+    _ req: DocumentHighlightRequest
+  ) async throws -> [DocumentHighlight]? {
+    nil
+  }
+
+  package func foldingRange(
+    _ req: FoldingRangeRequest
+  ) async throws -> [FoldingRange]? {
```

Comment:
Could all of these be on a single line instead of split? The function signatures look fairly short.

Reply:
I moved all the ones onto one line that swift-format would let me.